Lib/site.py

```diff
@@ -523,7 +523,12 @@ def register_readline():
             pass
 
         def write_history():
-            from _pyrepl.__main__ import CAN_USE_PYREPL
+            try:
+                # _pyrepl.__main__ is executed as the __main__ module
+                from __main__ import CAN_USE_PYREPL
+            except ImportError:
+                CAN_USE_PYREPL = True
```

Comment:
Shouldn't this probably be `False` if the import fails?

Reply:
I don't know. Let's use False on ImportError.